ilastik_feature_selection/wrapper_feature_selection.py

```diff
@@ -45,7 +44,7 @@ def kfold_train_and_predict(X, Y, classifier, k=5, indices=None, features=None):
         if features is None:
             features = np.array(list(range(X.shape[1])))
         features = np.array(list(features))
-        kf = cross_validation.KFold(len(indices), n_folds=k)
+        kf = model_selection.KFold(len(indices), n_folds=k)
         accurs = []
         for train, test in kf:
             train_ind = indices[train].astype("int")
```

Comment:
no need to go back to `indices` using the new interface, I assume

Reply:
Somewhat unnecessary, I see why they deprecated it...

Reply:
ah, sorry I missed that! I left the use of 'indices' for now, as they are also a parameter. This could easily be changed in the future...